### DISCUSSION TITLE
Expand current database from table list

### DISCUSSION THREAD

**Discussion by rajeshisnepali** [dbeaver/dbeaver#13765] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Every time I turn on my machine, & of course DBeaver. I've lots of tables opened. I've to go through the left sidebar & expand each host, database to get the actual database table. 

**Describe the solution you'd like**
Make an option (right-click) on the table (extend data) to go to the current database. Or at least save the settings, like how many tables are expanded in the sidebar before closing the app. 

**Describe alternatives you've considered**
NULL. Had to go through a long process to see all the tables. 

**Additional context**
![2021-08-31_11-28](https://user-images.githubusercontent.com/23610013/131448557-bcc3f749-9271-4f56-b5af-8e4af6145ce2.png)

**Comment by uslss** [comment]
Link with editor button is already implemented (you can see how it works in gif attached). Could you check if it resolves your problem?
![VmiXWG7qoa](https://user-images.githubusercontent.com/31996417/131460472-aaecf9fc-beae-4be6-b510-45297f28a8e3.gif)

**Comment by rajeshisnepali** [comment]
I can't find such a button in my DBeaver. My version information is
![image](https://user-images.githubusercontent.com/23610013/131612671-576062ed-7a93-49ec-ac99-e44faa1405b3.png)

![image](https://user-images.githubusercontent.com/23610013/131612737-ebed6c03-fc0c-4a2a-888b-8ab90e9017d7.png)
As you can see my top bar.

**Comment by ShadelessFox** [comment, marked as answer]
Hello @rajeshisnepali,

The cursor in this video has offset by some reason.

Here's the button:
![image](https://user-images.githubusercontent.com/35821147/131665480-7119bcb1-153b-4767-a513-60c3e04e376a.png)

**Comment by rajeshisnepali** [comment]
`Ctrl+Shift+,` is 🔥
